Problems with latest postcss-cli

Been tracing a build error I got this morning after merging in changes from our designer to add boot-autoprefixer. Looks like a new version of postcss-cli is out and requires a slight change in how files are referenced:

https://github.com/postcss/postcss-cli/issues/89

You end up with an error that looks like:

```
PluginError: Cannot find module '/home/david/.boot/cache/tmp/media/projects/report-center/fwy/6qp7pj/app.css'
```

Because the latest version interprets the arguments differently and treats your CSS as a plugin.

--- a/src/danielsz/autoprefixer.clj
+++ b/src/danielsz/autoprefixer.clj
@@ -24,11 +24,12 @@
       (doseq [[in-path in-file file] (find-css-files fileset files)]
         (boot.util/info "Autoprefixing %s\n" (:path file))
         (let [out-file (doto (io/file tmp-dir in-path) io/make-parents)
-              postcss (or exec-path "postcss")]
+              postcss (or exec-path "postcss")
+              args    (if browsers ["--autoprefixer.browsers" browsers] [])
+              args    (into args ["--" (.getPath in-file)])]
           (apply util/dosh postcss "--use" "autoprefixer"
-                 (.getPath in-file)
                  "-o" (.getPath out-file)
-                 (when browsers ["--autoprefixer.browsers" browsers]))))
+                 args)))
       (-> fileset
           (core/add-resource tmp-dir)
           core/commit!))))

